test/test_jit.py

```diff
@@ -14168,6 +14168,26 @@ def test():
         with self.assertRaisesRegex(Exception, "Parameters not specified"):
             torch.jit.script(test)
 
+        def wrong_decl_body(x: str) -> str:
```

Comment:
Nit:

Could you create a separate test case for it rather than extending existing test case? Long test case could make test failures less readable

Reply:
Another option is to use [subtests](https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests).